src/Community.VisualStudio.Toolkit.Shared/VS.cs

```diff
@@ -47,13 +48,25 @@ public static class VS
         public static async Task<TInterface> GetServiceAsync<TService, TInterface>() where TService : class where TInterface : class
```

Comment:
Now that this may return null, the nullable type ref annotation must be changed:

```suggestion
        public static async Task<TInterface?> GetServiceAsync<TService, TInterface>() where TService : class where TInterface : class
```

Reply:
Also the doc line above it uses incorrect syntax for the `null` keyword. It should be:

```diff
-/// <returns>A task who's result is the service, if found; otherwise <see langword="">null</see>.</returns>
+/// <returns>A task who's result is the service, if found; otherwise <see langword="null" />.</returns>